src/protocols/bitcoin/components/TransferSendForm.vue

```diff
@@ -185,7 +185,7 @@ export default defineComponent({
         0,
         formModel.value.address! || activeAccount.value.address,
         {
-          fee: 0,
+          fee: 0.00000100, // TODO: Insert actual fee here
```

Comment:
We don't need to set an actual fee here, we are calculating an actual fee below, judging by the `biteSize` that we are calculating constructing with an empty fee.